scripts/lib/logging.sh

```
@@ -64,11 +64,11 @@ log_with_timestamp() {
 show_progress() {
     local message="$1"
     local duration="${2:-3}"
-    
+
     for i in $(seq 1 "$duration"); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)